app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -292,12 +291,34 @@ public void startNextScenario() {
     public void run(View view) {
         circuitController.run();
         ((GridView)findViewById(R.id.circuit)).invalidateViews();
-        checkScenarioComplete();
+        checkScenarioComplete(true);
     }
 
-    private void checkScenarioComplete(){
+    private void checkScenarioComplete(boolean run){
```

Comment:
Wat doet deze boolean? Niet duidelijk, graag naam veranderen of erbij zetten in de comments!